.github/workflows/release.yaml

```diff
@@ -0,0 +1,91 @@
+# When pushing to main, we run checks and then release.
+
+name: Check and Release
+on:
+  push:
+    branches:
+    - main
+permissions:
+  contents: read
+env:
+  ARTIFACT_NAME: artifact-ubuntu-latest-3.10
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+    permissions:
+      contents: write # To create the release tag.
+    secrets: inherit
+
+  provenance:
+    needs: build
+    permissions:
+      actions: read   # To read the workflow path.
+      id-token: write # To sign the provenance.
+      contents: write # To add assets to a release.
+    # The generator should be referenced with a semantic version.
+    # The build will fail if we reference it using the commit sha.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.2.0
+    with:
+      base64-subjects: ${{ needs.build.outputs.artifacts-sha256 }}
+
+  release:
+    if: "!startsWith(github.event.commits[0].message, 'bump:')"
```

Comment:
Do we need the `"` around the expression here? It’s JS, isn’t it?

Reply:
The reason why this expression needs `"` is the `!` which is a reserved symbol in YAML.